15/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -0,0 +1,331 @@
+is executed and will run as PID 1. You can stop the detached container by running
+`podman stop postgresql_database`.
+
+
+
+Environment variables and volumes
+---------------------------------
+
+The image recognizes the following environment variables that you can set during
+initialization by passing `-e VAR=VALUE` to the Docker run command.
+
+**`POSTGRESQL_USER`**  
+       User name for PostgreSQL account to be created
+
+**`POSTGRESQL_PASSWORD`**  
+       Password for the user account
+
+**`POSTGRESQL_DATABASE`**  
+       Database name
+
+**`POSTGRESQL_ADMIN_PASSWORD`**  
+       Password for the `postgres` admin account (optional)
+
+
+Alternatively, the following options are related to migration scenario:
+
+**`POSTGRESQL_MIGRATION_REMOTE_HOST`**  
+       Hostname/IP to migrate from
+
+**`POSTGRESQL_MIGRATION_ADMIN_PASSWORD`**  
+       Password for the remote 'postgres' admin user
+
+**`POSTGRESQL_MIGRATION_IGNORE_ERRORS (optional, default 'no')`**  
+       Set to 'yes' to ignore sql import errors
+
+
+The following environment variables influence the PostgreSQL configuration file. They are all optional.
+
+**`POSTGRESQL_MAX_CONNECTIONS (default: 100)`**  
+       The maximum number of client connections allowed
+
+**`POSTGRESQL_MAX_PREPARED_TRANSACTIONS (default: 0)`**  
+       Sets the maximum number of transactions that can be in the "prepared" state. If you are using prepared transactions, you will probably want this to be at least as large as max_connections
+
+**`POSTGRESQL_SHARED_BUFFERS (default: 1/4 of memory limit or 32M)`**
+       Sets how much memory is dedicated to PostgreSQL to use for caching data
+
+**`POSTGRESQL_EFFECTIVE_CACHE_SIZE (default: 1/2 of memory limit or 128M)`**
+       Set to an estimate of how much memory is available for disk caching by the operating system and within the database itself
+
+
+You can also set the following mount points by passing the `-v /host/dir:/container/dir:Z` flag to Docker.
+
+**`/var/lib/pgsql/data`**  
+       PostgreSQL database cluster directory
+
+
+**Notice: When mouting a directory from the host into the container, ensure that the mounted
+directory has the appropriate permissions and that the owner and group of the directory
+matches the user UID or name which is running inside the container.**
+
+Typically (unless you use `podman run -u` option) processes in container
+run under UID 26, so -- on GNU/Linux -- you can fix the datadir permissions
+for example by:
+
+```
+$ setfacl -m u:26:-wx /your/data/dir
+$ podman run <...> -v /your/data/dir:/var/lib/pgsql/data:Z <...>
+```
+
+
+Data migration
+--------------
+
+PostgreSQL container supports migration of data from remote PostgreSQL server.
+You can run it like:
+
+```
+$ podman run -d --name postgresql_database \
+    -e POSTGRESQL_MIGRATION_REMOTE_HOST=172.17.0.2 \
+    -e POSTGRESQL_MIGRATION_ADMIN_PASSWORD=remoteAdminP@ssword \
+    [ OPTIONAL_CONFIGURATION_VARIABLES ]
+    openshift/postgresql-92-centos7
```

Comment:
Is the migration from `postgresql-92-centos7` really intended? What about some newer version?

Reply:
Good idea to update references in the doc. Will do.

Reply:
Done.